packages/react/src/UnorderedList/UnorderedListItem.tsx

```diff
@@ -0,0 +1,24 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import clsx from 'clsx'
+import { ForwardedRef, forwardRef, LiHTMLAttributes, PropsWithChildren } from 'react'
+
+export type UnorderedListItemProps = LiHTMLAttributes<HTMLLIElement>
+
+export const UnorderedListItem = forwardRef(
```

Comment:
Don't we want to use the same compound component structure Hee Chan suggested here?